### ISSUE TITLE
"Requested resource does not exist" when starting a live broadcast

### ISSUE THREAD

**Issue by dabe93** [mgp25/Instagram-API#2671]
### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question
- [ ] Other

---
Example Code:

```php
set_time_limit(0);
date_default_timezone_set('Europe/Rome');

require 'vendor/autoload.php';

class ExtendedInstagram extends \InstagramAPI\Instagram {

  public function changeUser(
    $username,
    $password)
    {
      $this->_setUser($username, $password);
    }

 public function finishChallenge(
   $response,
   $appRefreshInterval)
   {
     $this->_updateLoginState($response);
     $this->_sendLoginFlow(true, $appRefreshInterval);
   }

}

$ig = new ExtendedInstagram($debug, $truncatedDebug);
$loginResponse = $ig->login($username, $password);

$stream = $ig->live->create();
$broadcastId = $stream->getBroadcastId();

try {
   $ig->live->start($broadcastId);
} catch (\Exception $e) {
    echo 'Something went wrong: '.$e->getMessage()."\n";
    // exit(0);
}

echo "Enter 'stop' for terminate live streaming\n";
$line = trim(fgets($handler));

if ($line == 'stop'){
   $ig->live->end($broadcastId);
}
```

Error Log/var_dump:

```php
POST:  https://i.instagram.com/api/v1/live/17851288741436846/question_status/
← 404 	 0B
RESPONSE: 

Something went wrong: Requested resource does not exist.
```

---
### Describe your issue

Hi All,
after updating to the latest version I get the exception "Requested resource does not exist." when I try to start a live broadcast. The funny thing is that the broadcast is actually starting, so if I catch the exception without doing anything I can go on without issues, but I thing that is better to solve this issue.

Thank you

**Comment by JRoy** [comment]
Interesting, `question_status` may be using experiments, I will investigate this further.

**Comment by JRoy** [comment]
This should be fixed in 87b3d4960dfd2768385de4c7eeb12492124d46cd.

Cheers.
